features/password.feature

```diff
@@ -36,25 +37,53 @@ Feature: Using the installed keyring
         y
         """
         Then we should get no error
+        And we should not see the message "Failed to retrieve keyring"
         # @todo add step to check contents of keyring
 
     @todo
     Scenario: Open an encrypted journal with wrong password in keyring
     # This should ask the user for the password after the keyring fails
 
-    @todo
-    Scenario: Open encrypted journal when keyring exists but fails
-    # This should ask the user for the password after the keyring fails
-
     @todo
     Scenario: Decrypt journal with password in keyring
 
     @todo
     Scenario: Decrypt journal without a keyring
 
-    @todo
+    Scenario: Encrypt journal when keyring exists but fails
+        Given we use the config "simple.yaml"
+        And we have a failed keyring
+        When we run "jrnl --encrypt" and enter
+        """
+        karimpwnz
+        karimpwnz
```

Comment:
Can we set the password to something more semantic? It's unclear to me what's happening here. Maybe set it to something like `this password will not be saved`?

Reply:
Yes, let me do that!

Reply:
Done.